cmd/akamai/command.go

```diff
@@ -128,6 +128,7 @@ func RootCredentials() *cobra.Command {
 	authCmd.Flags().BoolVar(&copyArgoCDPasswordToClipboardFlag, "argocd", false, "copy the ArgoCD password to the clipboard (optional)")
 	authCmd.Flags().BoolVar(&copyKbotPasswordToClipboardFlag, "kbot", false, "copy the kbot password to the clipboard (optional)")
 	authCmd.Flags().BoolVar(&copyVaultPasswordToClipboardFlag, "vault", false, "copy the vault password to the clipboard (optional)")
+	authCmd.Flags().BoolVar(&ciFlag, "ci", false, "if running kubefirst in ci, set this flag to disable interactive features")
```

Comment:
PSA: you can create this flag on the root command as a `PersistentFlag` and it will be inherited by every subcommand. That way you don't have to remember to maintain it everywhere and just on a single place.